SocialStudies/SurviveTheMiddleAges/survive-the-middle-ages.ipynb

```diff
@@ -0,0 +1,377 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![Callysto.ca Banner](https://github.com/callysto/curriculum-notebooks/blob/master/callysto-notebook-banner-top.jpg?raw=true)\n",
+    "\n",
+    "<a href=\"https://hub.callysto.ca/jupyter/hub/user-redirect/git-pull?repo=https%3A%2F%2Fgithub.com%2Fcallysto%2Fcurriculum-notebooks&branch=master&subPath=SocialStudies/SurviveTheMiddleAges/survive-the-middle-ages.ipynb&depth=1\" target=\"_parent\"><img src=\"https://raw.githubusercontent.com/callysto/curriculum-notebooks/master/open-in-callysto-button.svg?sanitize=true\" width=\"123\" height=\"24\" alt=\"Open in Callysto\"/></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Survive the Middle Ages Game\n",
+    "\n",
+    "![sanvitale wikipedia](https://upload.wikimedia.org/wikipedia/commons/thumb/2/2c/Sanvitale03.jpg/320px-Sanvitale03.jpg)\n",
+    "\n",
+    "This is a game based on statistical mortality rates in Europe during the [Middle Ages](https://en.wikipedia.org/wiki/Middle_Ages). You will build a character and see how long your character, and their family, survives.\n",
+    "\n",
+    "You will need a way to keep track of what year it is as well as each character's age, gender, and marital status. Paper and pencil will work fine.\n",
+    "\n",
+    "\n",
+    "## Character Building\n",
+    "\n",
+    "### Country/Region\n",
+    "\n",
+    "Let's first look at the populations of countries or regions during this time. Select the following code cell and click the `►Run` button. It will download data from [Wikipedia](https://en.wikipedia.org/wiki/Medieval_demography) and create two charts."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "data_tables = pd.read_html('https://en.wikipedia.org/wiki/Medieval_demography')\n",
+    "for i in range(len(data_tables)):\n",
+    "    if data_tables[i].columns[0] == 'Country/Region':\n",
+    "        population = data_tables[i]\n",
+    "population.drop(population.index[[16, 17, 18]], inplace=True)\n",
+    "population = population.apply(pd.to_numeric, errors='ignore')\n",
+    "population['Average'] = population.mean(axis=1)\n",
+    "total_population = population['Average'].sum()\n",
+    "population['Percent'] = population['Average']/total_population*100\n",
+    "\n",
+    "!pip install plotly_express\n",
+    "import plotly_express as px\n",
```

Comment:
not sure why but only plotly.express worked on my local host. Not sure if this matters on hub